homework/length-sort/sort.cpp

```diff
@@ -0,0 +1,13 @@
+#include "sort.hpp"
+
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& list)
+{
+   
+  std::deque<std::string>round(list.begin(),list.end());
+    
+
+
+
+    return round;
+}
```

Comment:
can be in one line:
return std::deque<std::string>round(list.begin(),list.end());